dojo/templates/base.html

```diff
@@ -484,6 +478,9 @@ <h3 class="no-margin-top" style="padding-bottom: 5px;">
                       <li role="separator" class="divider"></li>
                       <li><a href="{% url 'add_endpoint' product_tab.product.id %}"><i class="fa fa-plus"></i> Add New Endpoint</a></li>
                       {% endif %}
+                      {% if product_tab.product|has_object_permission:"Import_Scan_Result" and 'ENDPOINT_META_IMPORTER'|setting_enabled %}
```

Comment:
`Endpoint_Edit` would be the right permission.

Reply:
Sure